src/pages/tickets/__tests__/Tickets.spec.tsx

```diff
@@ -5,6 +5,7 @@ import { mainStore } from 'store/main';
 import { uiStore } from 'store/ui';
 import { user } from '__test__/__mockData__/user';
 import Tickets from '../Tickets';
+import mockOrgBounties from '../../../bounties/__mock__/mockBounties.data';
 
```

Comment:
@okhot can you please remove the changes for TIckets.spec.tsx, since the test is solely for Organization Bounties, or is there a reason you added tests to Tickets.spec.tsx?